core/templates/google-analytics.initializer.ts

```diff
@@ -16,27 +16,40 @@
  * @fileoverview Initialization of Google Analytics (gtag.js).
  */
 
-import constants from 'assets/constants';
+declare const GA_ANALYTICS_ID: string;
```

Comment:
We need some way to ensure these don't pollute the global scope. Otherwise they run the risk of overwriting something else or being overwritten. Let's discuss offline.